api.js

```diff
@@ -76,7 +80,7 @@ apiRouter.get('/gltf', async (req, res) => {
         // Store the tid in Redis so we know that it's being processed; it will remain 'in-progress' until we
         // are notified that it is complete, at which point it will be the translation ID.
         if (resp.contentType.indexOf('json') >= 0) {
-            redisClient.set(JSON.parse(resp.data).id, 'in-progress');
+            inMemoryDataStore[JSON.parse(resp.data).id] = 'in-progress';
```

Comment:
Do we clear the ID from the  inMemoryDataStore? I assume we can do this once we have the results? If we don't this may grow and run out of memory.

Reply:
Functionality added